extensions/2.0/Khronos/KHR_animation2/README.md

```diff
@@ -0,0 +1,187 @@
+# KHR\_animation2
+
+## Contributors
+
+* Alexey Knyazev
+* Ben Houston, Threekit
+* Bryce Hutchings, Microsoft [@brycehutchings](https://twitter.com/brycehutchings)
+* Don McCurdy
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Jamie Marconi, Microsoft [@najadojo](https://twitter.com/najadojo)
+* Joe Herdman, [@jbherdman](https://github.com/jbherdman)
+* Lewis Weaver, Microsoft
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+Copyright (C) 2018-2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension is based on the animation features of glTF 2.0. The structure of the schemas stay the same.  
+  
+The only major addition is, that the output values are mapped using a [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901).
```

Comment:
I'd propose adding a snippet from the referenced spec here that I think is imporant:
> Evaluation of a JSON Pointer begins with a reference to the root
   value of a JSON document and completes with a reference to some value
   within the document.

The key point here being that one cannot move "upwards" in the path, a thing that I wondered about when reading this proposal - would it allow animating objects that are parallel or outside to the glTF scene? - with the answer from the spec being "no, needs to be within the document".